src/VisualStudio/Core/Def/Implementation/CodeLens/CodeLensCallbackListener.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Remote;
+using Microsoft.VisualStudio.Language.CodeLens;
+using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
+using Microsoft.VisualStudio.LanguageServices.Remote;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Settings;
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeLens
+{
+    /// <summary>
+    /// This is used by new codelens API to get extra data from VS if it is needed.
+    /// </summary>
+    [Export(typeof(ICodeLensCallbackListener))]
+    internal class CodeLensCallbackListener : ICodeLensCallbackListener
+    {
+        private const int DefaultMaxSearchResultsValue = 99;
```

Comment:
Strange number... why not 100?

Reply:
no idea why we landed on 100. tagging @Pilchie

Reply:
I think they don't want to show 3 digits.